lib/src/runner/browser/content_shell.dart

```diff
@@ -59,9 +60,16 @@ class ContentShell extends Browser {
         // make sure it's not expired and that the remote debugging port worked.
         // Any errors from this will always come before the "Running without
         // renderer sandbox" message.
-        while (await stderr.moveNext() &&
+        while (await stderr
+                .moveNext()
+                .timeout(_errorTimeout)
+                .catchError((_) => false) &&
             !stderr.current.endsWith("Running without renderer sandbox") &&
-            !stderr.current.contains("Running without the SUID sandbox")) {
+            !stderr.current.contains("Running without the SUID sandbox") &&
+            // Error messages on Mac can get gobbled, we must assume that it
+            // it started up successfully.
```

Comment:
Content shell starts up successfully without producing *any* output on standard error? Do certain versions do this reliably, or does it happen at random? In cases where this happens, how do we tell if `bind()` failed?

Assuming the answers are "yes", "at random", and "we can't", I guess we'll just work around it as best we can and hope we can remove content shell support soon. In that case, we probably want a shorter timeout—10s should be totally fine even on very slow systems.